cou/cli.py

```diff
@@ -125,10 +126,10 @@ async def analyze_and_plan(args: CLIargs) -> UpgradePlan:
     progress_indicator.succeed()
 
     progress_indicator.start("Generating upgrade plan...")
-    upgrade_plan = await generate_plan(analysis_result, args)
+    upgrade_plan, warning_messages = await generate_plan(analysis_result, args)
     progress_indicator.succeed()
```

Comment:
I think we should put progress_indicator failed if there are warning messages

Reply:
For planning, since generation of plan, the indicator should not be failed. For running upgrades, it is already being marked as failed.